doc/api/http.md

```diff
@@ -3206,6 +3206,41 @@ server.on('request', (request, res) => {
 server.listen(8000);
 ```
 
+## `http.createSimpleServer([directory[, port[, host]]])`
```

Comment:
Bikeshed: s/Simple/Static or s/Simple/File

Reply:
I'd prefer the argument here to be an `options` object that would allow us to more easily extend later, e,g,

```js
http.createStaticServer({
  root: '/the/directory',
  port: 1234,
  host: 'localhost'
})
```